docs/Design/List.md

```diff
@@ -0,0 +1,106 @@
+# List

```

Comment:
Let's make this a more specific design document than all of lists. You don't want to be in the business of defining how GLS describes them (which I separately filed as #129). Instead, call it something like "Native List Commands" (and rename the file accordingly).